README.md

```diff
@@ -57,7 +57,7 @@ REDIS_HOSTNAME | The hostname needed for connecting to Azure Redis | N/A
 REDIS_PORT | The port needed for connecting to Azure Redis | N/A
 REDIS_PASSWORD | The password needed for connecting to Azure Redis | N/A
 REDIS_PARTITION | The name of the application, needed for Azure Redis | ffc-grants-adult-cattle-housing-web
-SERVER_TIMEOUT | How long the server timeout should be when running loclaly | 5
+SERVER_TIMEOUT | How long the server timeout should be when running localy | 5
```

Comment:
Still a typo in 'localy'

Reply:
Actually looks like this SERVER_TIMEOUT is never used anywhere. I will remove it from README.